src/metabase/driver/mysql.clj

```diff
@@ -632,35 +632,45 @@
 
 (sql/register-clause! ::load format-load :insert-into)
 
+(defn- offset-datetime->unoffset-datetime
+  "Remove the offset from a datetime, returning a string represnetation in whatever timezone the `database` is configured
+  to use. This is necessary since MariaDB doesn't support timezone offsets and so we need to calculate one by hand."
```

Comment:
```suggestion
  to use. This is necessary since MariaDB doesn't support timestamp with time zone literals and so we need to calculate one by hand."
```
Technically the above suggestion is correct. MariaDB does support `CONVERT_TZ` for example, which kinda means it supports "timezone offsets"